proseco/guide.py

```diff
@@ -349,7 +367,7 @@ def get_candidates_mask(self, stars):
 
         """
         ok = ((stars['CLASS'] == 0) &
-              (stars['mag'] > 5.9) &
+              (stars['mag'] > 5.8) &
```

Comment:
Oops, had this fix local but neglected to push before your approval @taldcroft .